beekeeper-vacuum-tool/src/main/java/com/expediagroup/beekeeper/vacuum/BeekeeperVacuumToolApplication.java

```diff
@@ -179,6 +181,7 @@ private void removePath(Path toRemove, String databaseName, String tableName) {
           .tableName(tableName)
           .path(toRemove.toString())
           .housekeepingStatus(SCHEDULED)
+          .lifecycleType(UNREFERENCED.name())
```

Comment:
Can you write a unit test that asserts this?

Reply:
yes